hugo/content/docs/micro-layout.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Micro-layout
+weight: 400
+---
+
+The **micro-layout** refers to the layout of elements inside of a node, i.e. the layout of nested labels, buttons, etc. It is not to be confused with the **macro-layout** which refers to the layout of the entire graph, e.g. the position of nodes.
+
+## Layout Configuration
+Each element inheriting from `SNode` or `SCompartment` can have a property `layout` that specifies the layout of its children elements. Sprotty implements three layout configurations by default:
+* `hbox`: children elements are arranged horizontally
+* `vbox`: children elements are arranged vertically
+* `stack`: children elements are stacked on top of each other
+
+The [Class Diagram example](https://github.com/eclipse-sprotty/sprotty/tree/master/examples/classdiagram/) is a good starting point for understanding the layout configuration.
+Let's focus on `node0` which represents the class `Foo`:
+
+![node0](/micro-layout-node0.png)
```

Comment:
Would be cool to pimp that pic with some boxes (frames) around the respective areas to highlight the respective layout types.